lib/src/screens/home/feedTab.dart

```diff
@@ -12,8 +21,111 @@ class FeedTab extends StatefulWidget {
+      postDate: DateTime.now().add(Duration(hours: 2)),
+    ),
+  ];
+
   @override
   Widget build(BuildContext context) {
-    return Text("Base");
+    final size = MediaQuery.of(context).size;
+
+    return Container(
+      width: size.width,
+      height: size.height,
+      child: ListView.builder(
+        padding: EdgeInsets.all(20),
+        itemCount: feedItems.length,
+        itemBuilder: (context, i) => buildItem(context, feedItems[i]),
+      ),
+    );
+  }
+
+  Widget buildItem(BuildContext context, FeedItem item) {
+    VideoPlayerController? videoPlayerController;
+    // if (item.type == FeedItemType.Lecture && item.attachmentUrl != null) {
+    //   videoPlayerController =
+    //       VideoPlayerController.network(item.attachmentUrl!);
+    //   await videoPlayerController.initialize();
+    //}
+
+    return Padding(
+      padding: EdgeInsets.all(20),
+      child: Card(
+        child: InkWell(
+          onTap: () => {},
+          child: Container(
+            padding: EdgeInsets.all(5),
+            child: Column(
+              children: [
+                Row(
+                  children: [
+                    Text('${item.owner} posted ${item.name}'),
+                    Spacer(),
+                    Text('${postDateFormatter.format(item.postDate)}',
+                        style: TextStyle(color: Colors.grey[400])),
+                  ],
+                ),
+                SizedBox(height: 10),
+                Text(
+                  item.body,
+                  textAlign: TextAlign.center,
+                ),
+                SizedBox(height: 10),
+                SizedBox(height: 10),
+                Row(
+                  children: [
+                    if (item.type == FeedItemType.Homework)
+                      Row(
+                        children: [
+                          Icon(PlatformIcons(context).time),
+                          SizedBox(width: 5),
+                          Text('Appears in calendar',
+                              style: TextStyle(color: Colors.grey[450]))
+                        ],
+                      ),
+                    Spacer(),
+                    PlatformButton(
+                        onPressed: () => {},
+                        child: Row(
+                          children: [
+                            Icon(PlatformIcons(context).eyeSolid),
+                            SizedBox(width: 5),
+                            PlatformText('View'),
+                          ],
+                        ),
+                        color: Colors.white,
+                        cupertino: (context, platform) => CupertinoButtonData(
+                            color: AppColors.secondaryUofILight),
+                        materialFlat: (context, platform) =>
+                            MaterialFlatButtonData(
+                                textColor: Colors.white,
+                                color: AppColors.secondaryUofILight)),
```

Comment:
```suggestion
                        cupertino: (context, platform) => CupertinoButtonData(
                        color: MediaQuery.of(context).platformBrightness ==
                                Brightness.dark
                            ? AppColors.secondaryUofIDark
                            : AppColors.secondaryUofILight,
                      ),
                      materialFlat: (context, platform) =>
                          MaterialFlatButtonData(
                        textColor: Colors.white,
                        color: MediaQuery.of(context).platformBrightness ==
                                Brightness.dark
                            ? AppColors.secondaryUofIDark
                            : AppColors.secondaryUofILight,
                      ),
```
Change colors based on the device theme.

Reply:
This looks quite long and messy. I think it'd be good if the `PlatformButton` was refactored out so it was only one line.